### ISSUE TITLE
start minimized to tray

### ISSUE THREAD

**Issue by tom-vb** [danpla/dpscreenocr#19]
Hello,
Please allow me a suggestion. Would be nice an option to start the program minimized in the system tray. In my case the program starts when windows starts and I use it when I need it with a hotkey, I can minimize it every time in the system tray using some other program for this purpose, but it has to be done every time the session starts. An option for minimize itself to the system tray on startup would be wonderful.

A greeting. and congratulations for this fantastic program.

**Comment by danpla** [comment]
Hello,
I added a `ui_window_minimize_on_start` option (`false` by default) to the configuration file. If both `ui_tray_icon_visible` and `ui_window_minimize_to_tray` are enabled, the window will be hidden to tray.

**Comment by tom-vb** [comment]
Hi, many thanks! I will update right now...

**Comment by danpla** [comment]
You're welcome. Version 1.0.3 that will include this changes is not yet released, but you can build it manually from the most recent sources as deacribed in `[doc](https://github.com/danpla/dpscreenocr/tree/master/doc)/building-windows-msys2.txt

**Comment by tom-vb** [comment]
Yes, I saw it this morning. Thanks, but I will wait until the new version is released. I'm not familiar with C compilers...
